cms/static/js/models/settings/course_details.js

```diff
@@ -43,6 +45,12 @@ var CourseDetails = Backbone.Model.extend({
             }
             // TODO check if key points to a real video using google's youtube api
         }
+        if(_.has(newattrs, 'entrance_exam_minimum_score_pct')){
+            var intEntranceExamMinScore = Math.round(newattrs.entrance_exam_minimum_score_pct); // see if this ensures value saved is int
+            if (!isFinite(intEntranceExamMinScore) || /\D+/.test(newattrs.entrance_exam_minimum_score_pct) || intEntranceExamMinScore < 0 || intEntranceExamMinScore > 100) {
```

Comment:
I think this would be easier to read separated out into a helper method. As it stands I found it hard to be sure that the logic was correct. In fact, I could see this being a beneficial method in a number of places where we validate string values that should be integers.

FYI, I tested the field manually and all the edge cases I expected worked. I was surprised that 0% was a valid value but I figure that there are use cases where that might be useful.